java/src/jmri/jmrit/display/IconAdder.java

```diff
@@ -81,7 +81,7 @@ public IconAdder() {
         _userDefaults = false;
         _iconMap = new HashMap<>(10);
         _iconOrderList = new ArrayList<>();
-        this.setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        IconAdder.this.setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
```

Comment:
Why do you use `IconAdder.this.setLayout()` instead of `this.setLayout()`?

Reply:
It prevents the overridable method call in constructor warning.
This is an alternate method ( to creating a final / private method ) if it's just the 1 line which needs tweaking.
( First spotted in constructor comments for jmri.swing.NamedBeanComboBox L123)

Reply:
Yes, you are right. I didn't realised how that would work, but I now see that if this class is overridden, this line will still call the setLayout() in this class and not the setLayout() in the sub class.